generator/src/foreach_yarn

```diff
@@ -45,3 +49,14 @@ function mkdirSyncRecursive(dir) {
         curPath.length > 0 && !fs.existsSync(curPath) ? fs.mkdirSync(curPath) : null;
     }
 };
+
+function copyFolderSync(from, to) {
```

Comment:
hello, is it fast to copy all the plugins ?

Reply:
You mean, in comparison with calling system `cp -r` ?

Reply:
yes sorry I didn't write much comment

yes comparing this nodejs copy vs system cp

Reply:
Haven't checked yet. I can try to compare these two approach.

Reply:
Comparison provided in: https://github.com/eclipse/che-theia/pull/618#issuecomment-584079385